src/content/docs/apm/transactions/key-transactions/introduction-key-transactions.mdx

```diff
@@ -16,33 +16,230 @@ redirects:
+import apmKeyTransactionsTrackChanges from 'images/apm_screenshot-full_key-transactions-track-changes.png'
 
-For most New Relic agents, a key transaction can be any type of transaction. For the PHP, Node.js, and Go agents, only **web transactions** can be turned into key transactions.
+import apmServiceMapsandKeyTransactions from 'images/apm_screenshot-full_service-maps-and-key-transactions.png'
 
-## What are key transactions? [#overview]
+import apmLookoutandKeyTransactions from 'images/apm_screenshot-full_lookout-and-key-transactions.png'
 
-In APM, making a [transaction](/docs/apm/transactions/intro-transactions/transactions-new-relic-apm) a **key transaction** gives you the following benefits:
+import apmWorkloadsandKeyTransactions from 'images/apm_screenshot-full_workloads-and-key-transactions.png'
 
-* Key transactions are grouped together in the New Relic UI, so they are easy to find and examine.
-* You can [set a custom Apdex threshold](/docs/apm/transactions/key-transactions/create-key-transactions#apdex) for a key transaction.
-* You can set an [alert policy](/docs/apm/transactions/key-transactions/view-key-transactions-alert-information) for a key transaction.
+Key transactions are a way for engineering teams to create custom level monitoring on their most meaningful transactions. The APM monitoring tools of New Relic help you swiftly detect and solve problems before they impact your users by reporting on _all transactions_. But sometimes the information that is essential to running your digital business isn't the most highly trafficked.  This is where key transactions can help you track essential transactions you want to monitor and manage—regardless of the overall application throughput. 
 
-Key transactions are useful for:
+## Key transactions in practice 
 
-* **Important transactions.** For your app's most important transactions, making them key transactions means you can quickly find and examine them. You can set stricter Apdex requirements for them, and you can create alert policies for them.
-* **Slow transactions.** If a transaction is regularly slower than other transactions, it can bring down the Apdex score of the entire application. When you make a transaction a key transaction, you can set a longer-duration Apdex threshold for it so it won't bring down your app's Apdex score.
+Let's think about a scenario where a customer logs in to your e-commerce application to purchase a phone. The customer experiences slow response when they try load the plans page and browse the services. This results in the customer leaving the site and abandoning the purchase. Your `PlanService` microservice team needs to quickly identify the root cause of this slowdown and fix it to avoid losing more customers. 
```

Comment:
@homelessbirds I really like having this example that runs through the various sections!

Reply:
Thanks, rob!